src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.Spark;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+
+public class Intake extends SubsystemBase {
+
+  private final Spark m_IntakeMotor = new Spark(Constants.IntakeMotorPort);
+
+  private final DoubleSolenoid m_IntakeSolenoid = new DoubleSolenoid(Constants.IntakeSolenoidPort1,
+      Constants.IntakeSolenoidPort2);
+
+//Call these methods to set the motor direction for the intake
+  public void runIntakeMotorsForward() {
+    m_IntakeMotor.set(Constants.IntakeMotorSpeedScalar);
+  }
+
+  public void runIntakeMotorsBackwards() {
+    m_IntakeMotor.set(-Constants.IntakeMotorSpeedScalar);
+  }
+
+  public void runIntakeMotorsOff() {
+    m_IntakeMotor.set(0);
+  }
+
+//Use this to retract or extend the intake.
+  public void retractIntake() {
+    m_IntakeSolenoid.set(Value.kReverse);
+  }
+  public void extendIntake() {
+    m_IntakeSolenoid.set(Value.kForward);
+  }
+
+  /**
+   * Creates a new Intake.
+   */
+  //purpose?
+  public Intake() {
+
+  }
```

Comment:
This is the constructor. You don't need to use it in this case, but sometimes it can be useful. Read more about constructors here: [http://tutorials.jenkov.com/java/constructors.html](url)